### PR TITLE
READY : Removed redundant imports & moved implementation

### PR DESCRIPTION
**Pull request by JackieJoo**

### CODE REVIEW

File: proto/wtools/amid/color.test/Color.test.s [hunk 1]
```diff
@@ -5,11 +5,8 @@
 
 if( typeof module !== 'undefined' )
 {
-  // let _ = require( '../../../wtools/Tools.s' );
   let _ = require( '../../../wtools/Tools.s' );
-  require( '../color/l3/Color.s' );
-  require( '../color/l3/ColorCmyk.s' );
-  require( '../color/l3/ColorCmyka.s' );
+  _.include( 'wColor' )
```
Comment (Wandalen) [inline proto/wtools/amid/color.test/Color.test.s:9]: should be relative path and require

File: proto/wtools/amid/color/l3/ColorCmyk.s [hunk 2]
```diff
@@ -17,7 +17,6 @@
 if( typeof module !== 'undefined' )
```
Comment (Wandalen) [inline proto/wtools/amid/color/l3/ColorCmyk.s:17]: not include/requite on l3!

File: proto/wtools/amid/color/l3/ColorCmyk.s [hunk 3]
```diff
@@ -59,22 +55,31 @@ function _strToRgb( dst, src )
 
 function _longToRgb( dst, src )
 {
-  _.assert( src.length === 4, `{-src-} length must be 4, but got : ${src.length}` );
+  _.assert( src.length === 4 || src.length === 5, `{-src-} length must be 4 or 5, but got : ${src.length}` );
 
   let r, g, b;
+  let a = null;
```
Comment (Wandalen) [inline proto/wtools/amid/color/l3/ColorCmyk.s:61]: 1 by default!

File: proto/wtools/amid/color/l3/ColorCmyka.s [hunk 4]
```diff
@@ -149,7 +148,7 @@ function _formatStringParse( src )
 let Extension =
 {
 
-  // to rgb/a
+  // to rgba
```
Comment (Wandalen) [inline proto/wtools/amid/color/l3/ColorCmyka.s:151]: bad
_.assert( src.length === 5, `{-src-} length must be 5, but got : ${src.length}` );
should accept either 4 or 5!

Comment (JackieJoo) [reply proto/wtools/amid/color/l3/ColorCmyka.s:151]: It's cmyka namespace.
cmyk - accepts 4 or 5 
cmyka - accepts 5 only

File: proto/wtools/amid/color/l3/ColorCmyk.s (hunk 3 above)
Comment (JackieJoo) [reply proto/wtools/amid/color/l3/ColorCmyk.s:61]: let a = null;
is used to distinguish between cmyk and cmyka strings
it's later modified if alpha channel is present or remain null if not.

File: proto/wtools/amid/color/l3/ColorCmyk.s (hunk 2 above)
Comment (JackieJoo) [reply proto/wtools/amid/color/l3/ColorCmyk.s:17]: done

File: proto/wtools/amid/color.test/Color.test.s (hunk 1 above)
Comment (JackieJoo) [reply proto/wtools/amid/color.test/Color.test.s]: done

**Pull request merged**